src/lib/placeholders.test.ts

```diff
@@ -1,13 +1,13 @@
-import { containsPlaceholder } from './placeholders';
+import { containsNonArticleCountPlaceholder } from './placeholders';
```

Comment:
an extra test with `%%ARTICLE_COUNT%%` would be helpful

Reply:
yeah for sure! I'll add that now